### PR TITLE
Feature/open sessions shared

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: src/OpenSessionOverview/open-session-overview.less [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    }
+    .render-mobile & {
+      padding-top: 0;
+      max-width: 80%;
+    }
+  }
+}
+
+.session-tags {
+  display: flex;
```
Comment (tholex) [inline src/OpenSessionOverview/open-session-overview.less]: Seems like untouched display: block works just as well but ok :)

**Comment by tholex** [comment]
Stepped through the logic, makes sense. :+1:

**Comment by tholex** [comment]
Since the whole component is built via props, consider dropping it in a `Demo` component on the demo page

**Comment by learykara** [comment]
ah,  i always forget about demo. will do that

File: src/OpenSessionOverview/open-session-overview.less (hunk 1 above)
Comment (learykara) [inline src/OpenSessionOverview/open-session-overview.less]: was left over from when I copy-pasted - will update

**Pull request merged**